<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clearance Verification</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSuperElevation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Determine Super Elevation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBendRadius">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Determine Bend Radius</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture 2D Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Current Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBendRadius">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bend Radius:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSuperElevation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Super Elevation:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblExcess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Excess:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtSuperElevation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[1.23* Away]</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtBendRadius">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[123.45 Ft.]</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtExcess">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[12.34 In. (Center)]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHorizontalClearance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horiz. Clearance:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblVerticalClearance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vert. Clearance:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtHorizontalClearance">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[+123.45 In.]</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtVerticalClearance">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[+123.45 In.]</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmVisualizer">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>591</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Outside of Curve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>450</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inside of Curve -&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Images</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEquipmentOutside">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipment Side</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEquipmentInside">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipment Side</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtMQTTStatus">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>470</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MQTT Connection Lost</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actSave"/>
    <addaction name="separator"/>
    <addaction name="actReport"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actNew">
   <property name="text">
    <string>Create New</string>
   </property>
  </action>
  <action name="actReport">
   <property name="text">
    <string>Export Clearance Report</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
